src/_imagingcms.c

```diff
@@ -1511,15 +1511,16 @@ setup_module(PyObject *m) {
     PyObject *v;
     int vn;
 
-    CmsProfile_Type.tp_new = PyType_GenericNew;
```

Comment:
This is not needed (and incorrect! should call `core.profile_open` or `core.profile_frombytes` instead).

I added new tests for this being unset, but on PyPy this is the default value.